tutorials/schema_dataframes.ipynb

```diff
@@ -251,7 +251,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Commonly, you only tell Fugue: I want to create a dataframe, and here is the raw data source. And Fugue with certain execution engine will do the job for you. In you own logic, you mostly care about two abstract types in you functions: `DataFrame` and `LocalDataFrame`. They can be seen in the extentions tutorials."
+    "Commonly, you only tell Fugue: I want to create a dataframe, and here is the raw data source. And Fugue with certain execution engine will do the job for you. In your own logic, you mostly care about two abstract types in you functions: `DataFrame` and `LocalDataFrame`. They can be seen in the extension’s tutorials."
```

Comment:
Extensions is a section, so maybe we say "in the extensions section of the tutorials"